job-dsls/jobs/srcclr_scan_pipeline.groovy

```diff
@@ -45,8 +45,8 @@ parallel branches
         '''
 
 folder("${srcclrPath}")
-
-pipelineJob('srcclrpipeline') {
+def pipelineName = "${srcclrPath}/srcclrpipeline"
+pipelineJob(pipelineName) {
```

Comment:
why don't you change this by `pipelineJob("${srcclrPath}/srcclrpipeline")` and remove the `pipelineName` variable.